src/components/Header/Header.js

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 
-import { I18N } from '../../utils/constants';
-
 import style from './Header.module.css';
 
 const Header = (props) => {
+  const { t, i18n } = useTranslation();
   const { isAuth, login } = props;
+
+  const activateRu = (e) => {
+    i18n.changeLanguage('ru');
```

Comment:
Consider storing locale identifiers in the same I18N mentioned above.

Reply:
this including the default behavior of the library and is sufficiently convenient. Usually, libraries are used precisely for this purpose, so that without spending a lot of time, get the result

Reply:
It is possible to pass anything to the changeLanguage function - you're not using TypeScript.
Therefore, it make sense to store the locale (localisation string notation) in a specific place to control the naming for them in one place.